### ISSUE TITLE
Provided Schema does not match Table moz-fx-data-shar-nonprod-efed:telemetry_live.main_v4. Field payload.histograms.webrtc_hostname_obfuscation_disabled_ice_duration is missing in new schema, invalid

### ISSUE THREAD

**Issue by whd** [mozilla/mozilla-schema-generator#70]
/CC @fbertsch 

Looks like something got renamed: https://github.com/mozilla-services/mozilla-pipeline-schemas/commit/4d1226a54ec1c8b6c157bb76ec79c0e5e9a46115#diff-798e6758c033075932dae8f08d96c55bR11050.

**Comment by fbertsch** [comment]
Looking into this now

**Comment by fbertsch** [comment]
I've tracked this down to the fact that we're [using the current tip of `mozilla-central` when we run the probe-scraper.](https://github.com/mozilla/probe-scraper/blob/master/probe_scraper/scrapers/moz_central_scraper.py#L105) The problem is the tip changes the next day when it's run again.

What happens is this:
Day N: probe-scraper runs, scrapes the current tip, which has a new probe that is not present in any previous commits (to be clear, _not present in any commits that we scrape_, which is not all commits)
Day N+1: probe-scraper runs again, the tip has changed, and that new probe has been removed. Because we don't scrape _yesterday's_ tip, just today's, that probe is not present in any commit that we scrape at all. Thus it disappears from the probe-info-service.

Short-term, I'm going to explicitly add the commit for this new probe. Longer-term we need to stop using `json-tags` for scraping and only use the data in Buildhub; that way the probe-scraper is idempotent.
